csrc/transform_replay.cpp

```diff
@@ -1088,20 +1088,20 @@ void TransformPropagator::propagateSibling(TensorView* from, TensorView* to) {
   replayed_pos_[to] = pos;
 }
 
-TransformPropagator::TransformPropagator(TensorView* from, int64_t pos) {
+TransformPropagator::TransformPropagator(TensorView* from, int pos) {
```

Comment:
Why are we changing this to `int`? Shouldn't we use `int64_t` everywhere according to https://github.com/NVIDIA/Fuser/pull/82#discussion_r1150913183?

Reply:
I updated the functions in `transform_replay.cpp` to use `int64_t`